experimental/web/sample_dynamic/build_sample.sh

```diff
@@ -14,8 +14,29 @@
 #
 # Usage:
 #   build_sample.sh (optional install path) && serve_sample.sh
+#
+# The desired host install directory can be passed as the first argument.
+# Otherwise, it looks for an install directory under path set in the environment
+# variable IREE_HOST_BUILD_DIR (default build-host). The build directory for the
+# empscripten build is taken from the environment variable
+# IREE_EMPSCRIPITEN_BUILD_DIR, defaulting to "build-empscripten". Designed for
```

Comment:
Typos, these should be "emscripten", not "empscripten" or "EMPSCRIPITEN"

https://emscripten.org/

Reply:
Wow pretty impressive that I managed to misspell this in two different ways.